### ISSUE TITLE
Category search for Playground no longer possible

### ISSUE THREAD

**Issue by gvSmartEnergy** [osmandapp/OsmAnd#20508]
### Description

When entering Playground in search, the main category of Playground is not shown (instead various subtypes). An attempt to search from any given POI Playground for more fails as well. Restart did not change. 

### Steps to reproduce

Search for Playground category. 

### Actual result

Not selectable. 

### Expected result

Category selectable on top and nearby search shows all on map. 

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version:
Android/iOS version:
Device model:
Crash-Logs: ?
```

**Comment by yuriiurshuliak** [comment]
The bug was not reproduced. To reproduce the steps, open Search → Categories and search for the Playground category. The search was successful, and the category was found and displayed on the map.

So, please provide more details from your side and share a video demonstrating how it plays out for you.

OsmAnd~ 4.9.0#2909m, released: 2024-08-06

https://github.com/user-attachments/assets/c8bb55ed-ca35-4817-af82-29f5b0e41f9e

**Comment by gvSmartEnergy** [comment]
I checked for updates. Here a recording. 
Uploading signal-2024-08-06-20-13-36-148.mp4…

**Comment by gvSmartEnergy** [comment]
Checked for updates. Here a video. Thanks. 
https://github.com/user-attachments/assets/4fe2636f-64a3-41be-aa95-1dac4894dce4

**Comment by gvSmartEnergy** [comment]
Hi, thanks. It was OK. As soon as I synced the cloud data it reappeared. I have no proper data the next days, I wil try to figure out which file did it.

**Comment by scosu** [comment]
I think I can reproduce this issue quite reliably:

- Reset the app by clearing the app storage
- Start the app, skip downloading
- Go to map resources and download Europe -> Germany -> Standard map
- Wait for the Download to finish
- Stop the app
- Start the app again and search for playground

After these steps in most cases I have Playground being suggested in Tourism then.
This doesn't happen if I just download the single state that I am in, e.g. Berlin.

**Comment by gvSmartEnergy** [comment]
Hi. Sorry. I just waited and it indeed stopped. The latest version fixed a lot of issues on Pixel GrapheneOS including sizes of icons etc. Thanks. Sent from phone, please excuse typos. From: Markus Schneider-Pargmann ***@***.***>Sent: Saturday, December 28, 2024 3:32:45 pmTo: osmandapp/OsmAnd ***@***.***>Cc: gvSmartEnergy ***@***.***>; Author ***@***.***>Subject: Re: [osmandapp/OsmAnd] Category search for Playground no longer possible (Issue #20508)
I think I can reproduce this issue quite reliably:

Reset the app by clearing the app storage
Start the app, skip downloading
Go to map resources and download Europe -> Germany -> Standard map
Wait for the Download to finish
Stop the app
Start the app again and search for playground

After these steps in most cases I have Playground being suggested in Tourism then.
This doesn't happen if I just download the single state that I am in, e.g. Berlin.

**Comment by scosu** [comment]
For me this issue still happens after every reset of the app. The steps above seem to reliably trigger it for me, but it also happens after a while without executing those steps. The Playgrounds show up as Tourism and nothing is found on the map then.

**Comment by scosu** [comment]
This issue still happens for me reliably with the same steps to reproduce it as documented above

> I think I can reproduce this issue quite reliably:
> 
>     * Reset the app by clearing the app storage
> 
>     * Start the app, skip downloading
> 
>     * Go to map resources and download Europe -> Germany -> Standard map
> 
>     * Wait for the Download to finish
> 
>     * Stop the app
> 
>     * Start the app again and search for playground
> 
> 
> After these steps in most cases I have Playground being suggested in Tourism then. This doesn't happen if I just download the single state that I am in, e.g. Berlin.

I found a workaround by using Search -> Categories -> Custom search -> Leisure -> Playground but it is still really annoying and obviously a bug. Can you please reopen this?

@vshcherb @yuriiurshuliak

**Comment by DmitryAlexei** [comment]
@scosu Have you tried to reinstall the app (and redownload map data)?
Before reinstalling, please back up all personal data (tracks and favorites) by going to Menu -> Settings -> Export to file.

**Comment by scosu** [comment]
@DmitryAlexei As documented in the steps to reproduce the issue, I always clear all app data which completely resets the app. This should be equivalent to reinstalling the app.

**Comment by DmitryAlexei** [comment]
The issue might have been fixed here https://github.com/osmandapp/OsmAnd/issues/22824 . The issue was caused by specific German maps (not all). 

Fix is available in the nightly version of the app and soon will be available in the Play Store version.

**Comment by isso-git** [comment]
@scosu Hello scosu, as mentioned above.
I am realy sure this issue is fixed with Osmand **5.1.8**, because I can reproduce before and tested after with whole maps of Germany.
In versions before **5.1.8** the issue is present. 
Also in nightly builds after 15.07.25 it should work, there i have tested it with some affected maps.

**Comment by scosu** [comment]
Thank you, I am still running on 5.1.7 here. I will retest this once I have 5.1.8. Thanks for fixing!

**Comment by scosu** [comment]
I just tried to reproduce and this issue seems to be solved for me on 5.1.9. Thanks for fixing!
